plugin/pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -97,6 +97,7 @@ func (p *podAffinityPriorityMap) processTerm(term *api.PodAffinityTerm, podDefin
 				if p.failureDomains.NodesHaveSameTopologyKey(node, fixedNode, term.TopologyKey) {
 					p.counts[node.Name] += weight
 				}
+				// TODO i think we need to  add processing logic for pod labels here?
```

Comment:
You know what?  I think this is really handled at the node level.

The bug here is that the test is expecting this to be handled under *interpod_affinity* priorities, but really, its handled as part of the *node_affinity* logic.

Does that sound about right?  If so i think its a one line fix :)